### PR TITLE
Update for Bio-Formats 6.5.0

### PR DESCRIPTION
**Pull request by dgault**
Add release announcement and bump for Bio-Formats 6.5.0
See https://trello.com/c/5vKr2j9X/20-open-pr-against-wwwopenmicroscopyorg

### CODE REVIEW

File: _posts/2020-04-30-bio-formats-6-5-0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  * added support for single site HCS variant
+* Zeiss CZI
+  * fixed the position count when only one position is present and the starting index is greater than 0
+* Various Readers
+  * reviewed and updated readers to prevent potential cases of integer overflow
+
+OME-Model updates:
+
+* version of ``OME-Model`` has been updated to 6.1.0
+* added support for Python 3.8 and make code-generation Python 3 only (thanks to Roger Leigh)
```
Comment (mtbc) [inline _posts/2020-04-30-bio-formats-6-5-0.md:34]:
```suggestion
* added support for Python 3.8 and made code-generation Python 3 only (thanks to Roger Leigh)
```

**Comment by pwalczysko** [comment]
Looks and build well, see https://snoopycrimecop.github.io/www.openmicroscopy.org/2020/04/30/bio-formats-6-5-0.html

**Comment by pwalczysko** [comment]
Do not merge yet, see https://openmicroscopy.slack.com/archives/C0K5EED3R/p1588238762296800 @joshmoore @sbesson

**Comment by pwalczysko** [comment]
After this morning call discussion, merging now.

**Pull request merged**